doc/source/index.rst

```diff
@@ -1,4 +1,62 @@
-.. include:: ../../README.rst
+PyAnsys Actions
+===============
+|pyansys| |MIT|
+
+.. |pyansys| image:: https://img.shields.io/badge/Py-Ansys-ffc107.svg?logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAIAAACQkWg2AAABDklEQVQ4jWNgoDfg5mD8vE7q/3bpVyskbW0sMRUwofHD7Dh5OBkZGBgW7/3W2tZpa2tLQEOyOzeEsfumlK2tbVpaGj4N6jIs1lpsDAwMJ278sveMY2BgCA0NFRISwqkhyQ1q/Nyd3zg4OBgYGNjZ2ePi4rB5loGBhZnhxTLJ/9ulv26Q4uVk1NXV/f///////69du4Zdg78lx//t0v+3S88rFISInD59GqIH2esIJ8G9O2/XVwhjzpw5EAam1xkkBJn/bJX+v1365hxxuCAfH9+3b9/+////48cPuNehNsS7cDEzMTAwMMzb+Q2u4dOnT2vWrMHu9ZtzxP9vl/69RVpCkBlZ3N7enoDXBwEAAA+YYitOilMVAAAAAElFTkSuQmCC
+   :target: https://docs.pyansys.com/
+   :alt: PyAnsys
+
+.. |MIT| image:: https://img.shields.io/badge/License-MIT-yellow.svg
+   :target: https://opensource.org/licenses/MIT
+   :alt: MIT
+
+A repository containing a collection of `GitHub Workflows
+<https://docs.github.com/en/actions/using-workflows/about-workflows>`_ to be
+reused by projects in the PyAnsys ecosystem.
+
+For more information on available actions and how to use them,
+see each card below:
+
+.. grid:: 1 1 2 2
+    :gutter: 2
+
+    .. grid-item-card:: :octicon:`codescan-checkmark` Style actions
+       :link: style-actions/index
+       :link-type: doc
+
+        Style actions verify code and documentation.
+
+    .. grid-item-card:: :octicon:`tools` Tests actions
+       :link: tests-actions/index
+       :link-type: doc
+
+        The tests actions allow to run the test suite.
+
+    .. grid-item-card:: :octicon:`book` Docs actions
+       :link: doc-actions/index
+       :link-type: doc
+
+        Documentation actions build and deploy the documentation.
+
+    .. grid-item-card:: :octicon:`key` Licence actions
+       :link: licenses-actions/index
+       :link-type: doc
+
+        Description of actions related with the licenses.
+
+
+    .. grid-item-card:: :octicon:`package` Build actions
+       :link: build-actions/index
+       :link-type: doc
+
+        The build actions allow for building artifacts.
+
+    .. grid-item-card:: :octicon:`eye` Release actions
+       :link: release-actions/index
+       :link-type: doc
+
+        Release actions provide for releasing the various artifacts.
```

Comment:
Try to make the descriptions look uniform:

* Always start with "``ACTION_NAME`` enable repository workflows to ``WHATEVER``"